src/components/toast/components/toast-display/index.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames'
+import { ToastColor, ToastDisplayProps } from './types'
+import CloseButton from '../close-button'
+import s from './toast-display.module.css'
+
+const IS_DEV = process.env.NODE_ENV !== 'production'
+
+/**
+ * Display component for use with a toast library,
+ * such as react-hot-toast.
+ */
+function ToastDisplay({
+  title,
+  description,
+  color = ToastColor.neutral,
+  icon,
+  children,
+  renderActions,
+  dismissSelf,
+}: ToastDisplayProps) {
+  /**
+   * In development, throw an error if a toast has
+   * no title and no description (except when using custom content).
+   */
+  if (IS_DEV && !title && !description && !children) {
+    throw new Error(
+      `Toast must be provided either a "title" or a "description", or you must use "children" to render custom content in the Toast. Please ensure toast has a non-empty string for its "title" or "description", or provide custom "children" content to render.`
+    )
+  }
+
+  const DescriptionElem =
+    typeof description === 'string' ? 'p' : description ? 'div' : null
```

Comment:
🚧 I think we should only allow a `string` `description` for now, so it's not so easy to render something inaccessible.

Reply:
✅ Have pared this back to only support a plain text string for `description`